### DISCUSSION TITLE
@wechatferry/puppet 的构建系统好像有问题

### DISCUSSION THREAD

**Discussion by yArna** [wechatferry/wechatferry#7] [General]
@wechatferry/puppet
这个包的 `dist/index.mjs` 的运行逻辑与 `src/index.ts` 下的不一样

看了下  @wechatferry/puppet/src/events/index.ts 中的代码，在 `dist/index.mjs` 不存在，比如 Friendship 的就没有

这导致直接用 @wechatferry/puppet 包，不会触发  friendship 事件，而如果 `import { WechatferryPuppet } from "@wechatferry/puppet/src";` 就可以触发

**Comment by KeJunMao** [comment]
刚刚 build 了下果真如此，monorepo 害人不浅，似乎是没有 export 的的问题，我看看怎处理

**Comment by KeJunMao** [comment]
已修复，稍晚一些会发个 release

**Reply by zhao-heng-sheng** [reply]
在v0.0.23版本也有同样的问题
